poprox-db/migrations/versions/2025_02_06_1339-6da1373105ef_add_column_for_qualtrics_question_.py

```diff
@@ -0,0 +1,26 @@
+"""add column for qualtrics question metadata
+
+Revision ID: 6da1373105ef
+Revises: a67fedcb0af0
+Create Date: 2025-02-06 13:39:14.630988
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import JSONB
+
+# revision identifiers, used by Alembic.
+revision: str = "6da1373105ef"
+down_revision: Union[str, None] = "a67fedcb0af0"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.add_column("qualtrics_surveys", sa.Column("question_metadata", JSONB, nullable=True))
```

Comment:
I think there's a discussion to be had here around what we name this column. I think there's an argument for `question_metadata_raw` I think a lot of similar database columns have `_raw` as part of the name.

Open to change, looking for a second oppinion